ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/shared/target/messages.properties

```diff
@@ -144,4 +144,9 @@ UpdateTargetJob_UpdatingTarget=Updating Target
 EditTargetContainerPage_Add_Title=Create target reference
 EditTargetContainerPage_Edit_Title=Edit target reference
 EditTargetContainerPage_Message=Please enter an URI to the target to be referenced
-EditTargetContainerPage_Not_A_File={0} is not a file
\ No newline at end of file
+EditTargetContainerPage_Not_A_File={0} is not a file
+EditRepositoryContainerPage_Message=Please enter an URI that points to the repository
```

Comment:
I think it should be `Please enter a URI that points to the repository`
Similar to `a university` instead of `an university`. 

Other locations in this properties file use the wrong `an URI` variant, too.

Reply:
Fixed